config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "home#index"
+  resources :businesses
+  resources :cities
+  resources :categories
```

Comment:
Using `resources` like this creates a bunch of routes right for each "entity"? Are you using them now? I would recommend to only add what you're using.